scvi/module/_classifier.py

```diff
@@ -45,20 +45,27 @@ def __init__(
     ):
         super().__init__()
         self.logits = logits
-        layers = [
-            FCLayers(
-                n_in=n_input,
-                n_out=n_hidden,
-                n_layers=n_layers,
-                n_hidden=n_hidden,
-                dropout_rate=dropout_rate,
-                use_batch_norm=use_batch_norm,
-                use_layer_norm=use_layer_norm,
-                activation_fn=activation_fn,
-                **kwargs,
-            ),
-            nn.Linear(n_hidden, n_labels),
-        ]
+        layers = []
+
+        if n_hidden > 0 and n_layers > 0:
```

Comment:
can you document this in this class too?

Reply:
done